src/wallet/test/load_util.cpp

```
@@ -0,0 +1,54 @@
+#include <memory>
+#include <optional>
+#include <vector>
+
+namespace wallet {
+std::shared_ptr<CWallet> TestLoadWallet(std::unique_ptr<WalletDatabase> database, WalletContext& context, uint64_t create_flags)
+{
+    bilingual_str error;
+    std::vector<bilingual_str> warnings;
+    auto wallet = CWallet::Create(context, "", std::move(database), create_flags, error, warnings);
+    if (context.coinjoin_loader) {
+        // TODO: see CreateWalletWithoutChain
```

Cppcheck (2.21.0)

[error] `#error` No known always_inline attribute for this platform.

(preprocessorErrorDirective)